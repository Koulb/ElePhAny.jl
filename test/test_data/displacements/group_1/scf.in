&CONTROL
   calculation      = 'scf'
   verbosity        = 'high'
   tstress          = .false.
   tprnfor          = .true.
   outdir           = './tmp/'
   prefix           = 'scf'
   pseudo_dir       = './tmp/scf.save/'
/
&SYSTEM
   ibrav            = 2
   nbnd             = 32
   ecutwfc          = 60
   nosym            = .false.
   noinv            = .false.
   ntyp             = 1
   nat              = 16
   celldm(1)        = 20.524391127487448
/
&ELECTRONS
   electron_maxstep = 1000
   conv_thr         = 1e-13
   mixing_mode      = 'plain'
   mixing_beta      = 0.7
/
&IONS
/
&CELL
/

ATOMIC_SPECIES
Si 28.0855 Si.upf

K_POINTS automatic
1 1 1  0 0 0

ATOMIC_POSITIONS crystal
Si 0.0000689040 -0.0000000000 -0.0000000000 
Si 0.5000000000 0.0000000000 0.0000000000 
Si 0.0000000000 0.5000000000 -0.0000000000 
Si 0.5000000000 0.5000000000 -0.0000000000 
Si -0.0000000000 0.0000000000 0.5000000000 
Si 0.5000000000 0.0000000000 0.5000000000 
Si -0.0000000000 0.5000000000 0.5000000000 
Si 0.5000000000 0.5000000000 0.5000000000 
Si 0.3750000000 0.3750000000 0.3750000000 
Si 0.8750000000 0.3750000000 0.3750000000 
Si 0.3750000000 0.8750000000 0.3750000000 
Si 0.8750000000 0.8750000000 0.3750000000 
Si 0.3750000000 0.3750000000 0.8750000000 
Si 0.8750000000 0.3750000000 0.8750000000 
Si 0.3750000000 0.8750000000 0.8750000000 
Si 0.8750000000 0.8750000000 0.8750000000 

