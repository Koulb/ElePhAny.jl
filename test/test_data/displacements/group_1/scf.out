
     Program PWSCF v.7.2 starts on  4Jul2024 at 15:49:39 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     8 processors

     MPI processes distributed on     1 nodes
     16916 MiB available memory on the printing compute node when the environment starts
 
     Reading input from scf.in
Warning: card &CELL ignored
Warning: card / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4
 
     R & G space division:  proc/nbgrp/npool/nimage =       8
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         432     432    107                16935    16935    2110
     Max         433     433    108                16941    16941    2113
     Sum        3463    3463    859               135515   135515   16889
 
     Using Slab Decomposition
 


     bravais-lattice index     =            2
     lattice parameter (alat)  =      20.5244  a.u.
     unit-cell volume          =    2161.4782 (a.u.)^3
     number of atoms/cell      =           16
     number of atomic types    =            1
     number of electrons       =        64.00
     number of Kohn-Sham states=           32
     kinetic-energy cutoff     =      60.0000  Ry
     charge density cutoff     =     240.0000  Ry
     scf convergence threshold =      1.0E-13
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     celldm(1)=  20.524391  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.500000   0.000000   0.500000 )  
               a(2) = (   0.000000   0.500000   0.500000 )  
               a(3) = (  -0.500000   0.500000   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.000000 -1.000000  1.000000 )  
               b(2) = (  1.000000  1.000000  1.000000 )  
               b(3) = ( -1.000000  1.000000 -1.000000 )  


     PseudoPot. # 1 for Si read from file:
     /home/apolyukhin/Development/frozen_phonons/elph/example/pseudo/Si.upf
     MD5 check sum: 02fab3f35e82123ef5bf1cb05d5b1a5e
     Pseudo is Norm-conserving + core correction, Zval =  4.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1510 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2

     atomic species   valence    mass     pseudopotential
        Si             4.00    28.08550     Si( 1.00)

      2 Sym. Ops. (no inversion) found


                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym =  2     inv. 180 deg rotation - cart. axis [1,0,1]   

 cryst.   s( 2) = (     1          0          0      )
                  (     0          0          1      )
                  (     0          1          0      )

 cart.    s( 2) = (  0.0000000  0.0000000 -1.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )


     point group C_s (m)    
     there are  2 classes
     the character table:

       E     s    
A'     1.00  1.00
A''    1.00 -1.00

     the symmetry operations in each class and the name of the first element:

     E        1
          identity                                               
     s        2
          inv. 180 deg rotation - cart. axis [1,0,1]             

   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Si  tau(   1) = (  -0.0000345   0.0000000   0.0000345  )
         2           Si  tau(   2) = (  -0.2500000   0.0000000   0.2500000  )
         3           Si  tau(   3) = (   0.0000000   0.2500000   0.2500000  )
         4           Si  tau(   4) = (  -0.2500000   0.2500000   0.5000000  )
         5           Si  tau(   5) = (  -0.2500000   0.2500000   0.0000000  )
         6           Si  tau(   6) = (  -0.5000000   0.2500000   0.2500000  )
         7           Si  tau(   7) = (  -0.2500000   0.5000000   0.2500000  )
         8           Si  tau(   8) = (  -0.5000000   0.5000000   0.5000000  )
         9           Si  tau(   9) = (  -0.3750000   0.3750000   0.3750000  )
        10           Si  tau(  10) = (  -0.6250000   0.3750000   0.6250000  )
        11           Si  tau(  11) = (  -0.3750000   0.6250000   0.6250000  )
        12           Si  tau(  12) = (  -0.6250000   0.6250000   0.8750000  )
        13           Si  tau(  13) = (  -0.6250000   0.6250000   0.3750000  )
        14           Si  tau(  14) = (  -0.8750000   0.6250000   0.6250000  )
        15           Si  tau(  15) = (  -0.6250000   0.8750000   0.6250000  )
        16           Si  tau(  16) = (  -0.8750000   0.8750000   0.8750000  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           Si  tau(   1) = (  0.0000689  0.0000000  0.0000000  )
         2           Si  tau(   2) = (  0.5000000  0.0000000  0.0000000  )
         3           Si  tau(   3) = (  0.0000000  0.5000000  0.0000000  )
         4           Si  tau(   4) = (  0.5000000  0.5000000  0.0000000  )
         5           Si  tau(   5) = (  0.0000000  0.0000000  0.5000000  )
         6           Si  tau(   6) = (  0.5000000  0.0000000  0.5000000  )
         7           Si  tau(   7) = (  0.0000000  0.5000000  0.5000000  )
         8           Si  tau(   8) = (  0.5000000  0.5000000  0.5000000  )
         9           Si  tau(   9) = (  0.3750000  0.3750000  0.3750000  )
        10           Si  tau(  10) = (  0.8750000  0.3750000  0.3750000  )
        11           Si  tau(  11) = (  0.3750000  0.8750000  0.3750000  )
        12           Si  tau(  12) = (  0.8750000  0.8750000  0.3750000  )
        13           Si  tau(  13) = (  0.3750000  0.3750000  0.8750000  )
        14           Si  tau(  14) = (  0.8750000  0.3750000  0.8750000  )
        15           Si  tau(  15) = (  0.3750000  0.8750000  0.8750000  )
        16           Si  tau(  16) = (  0.8750000  0.8750000  0.8750000  )

     number of k points=     1
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000

     Dense  grid:   135515 G-vectors     FFT dimensions: (  72,  72,  72)

     Dynamical RAM for                 wfc:       1.04 MB

     Dynamical RAM for     wfc (w. buffer):       1.04 MB

     Dynamical RAM for           str. fact:       0.26 MB

     Dynamical RAM for           local pot:       0.00 MB

     Dynamical RAM for          nlocal pot:       9.32 MB

     Dynamical RAM for                qrad:       1.24 MB

     Dynamical RAM for          rho,v,vnew:       1.84 MB

     Dynamical RAM for               rhoin:       0.61 MB

     Dynamical RAM for            rho*nmix:       4.14 MB

     Dynamical RAM for           G-vectors:       1.10 MB

     Dynamical RAM for          h,s,v(r/c):       0.19 MB

     Dynamical RAM for          <psi|beta>:       0.14 MB

     Dynamical RAM for                 psi:       2.07 MB

     Dynamical RAM for                hpsi:       2.07 MB

     Dynamical RAM for      wfcinit/wfcrot:       4.23 MB

     Estimated static dynamical RAM per process >      17.71 MB

     Estimated max dynamical RAM per process >      26.31 MB

     Estimated total dynamical RAM >     210.49 MB

     Initial potential from superposition of free atoms

     starting charge      63.9902, renormalised to      64.0000
     Starting wfcs are   64 randomized atomic wfcs

     total cpu time spent up to now is        0.3 secs

     Self-consistent Calculation

     iteration #  1     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           103 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16671 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.00E-02,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.5 secs

     total energy              =    -133.84060738 Ry
     estimated scf accuracy    <       0.71593584 Ry

     iteration #  2     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           109 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16614 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.12E-03,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.6 secs

     total energy              =    -133.88043968 Ry
     estimated scf accuracy    <       0.05537710 Ry

     iteration #  3     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           113 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16606 MiB available memory on the node where the printing process lives
------------------
     ethr =  8.65E-05,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.8 secs

     total energy              =    -133.89317225 Ry
     estimated scf accuracy    <       0.00071689 Ry

     iteration #  4     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           114 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16603 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.12E-06,  avg # of iterations =  6.0

     total cpu time spent up to now is        1.0 secs

     total energy              =    -133.89466795 Ry
     estimated scf accuracy    <       0.00008584 Ry

     iteration #  5     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           115 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16599 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.34E-07,  avg # of iterations =  8.0

     total cpu time spent up to now is        1.2 secs

     total energy              =    -133.89470023 Ry
     estimated scf accuracy    <       0.00002723 Ry

     iteration #  6     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           116 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16607 MiB available memory on the node where the printing process lives
------------------
     ethr =  4.25E-08,  avg # of iterations =  5.0

     total cpu time spent up to now is        1.4 secs

     total energy              =    -133.89470807 Ry
     estimated scf accuracy    <       0.00000133 Ry

     iteration #  7     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           116 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16608 MiB available memory on the node where the printing process lives
------------------
     ethr =  2.08E-09,  avg # of iterations =  7.0

     total cpu time spent up to now is        1.6 secs

     total energy              =    -133.89470878 Ry
     estimated scf accuracy    <       0.00000064 Ry

     iteration #  8     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           117 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16601 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.00E-09,  avg # of iterations =  5.0

     total cpu time spent up to now is        1.7 secs

     total energy              =    -133.89470887 Ry
     estimated scf accuracy    <       0.00000012 Ry

     iteration #  9     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           118 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16606 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.81E-10,  avg # of iterations =  8.0

     total cpu time spent up to now is        1.9 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <       0.00000001 Ry

     iteration # 10     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           118 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16601 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.79E-11,  avg # of iterations =  8.0

     total cpu time spent up to now is        2.1 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          4.5E-09 Ry

     iteration # 11     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           119 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16601 MiB available memory on the node where the printing process lives
------------------
     ethr =  7.05E-12,  avg # of iterations =  6.0

     total cpu time spent up to now is        2.2 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          6.7E-10 Ry

     iteration # 12     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           119 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16601 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.05E-12,  avg # of iterations =  7.0

     total cpu time spent up to now is        2.5 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          1.8E-10 Ry

     iteration # 13     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           119 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16600 MiB available memory on the node where the printing process lives
------------------
     ethr =  2.76E-13,  avg # of iterations =  5.0

     total cpu time spent up to now is        2.6 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          4.5E-11 Ry

     iteration # 14     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           119 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16596 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.00E-13,  avg # of iterations =  2.0

     total cpu time spent up to now is        2.8 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          4.0E-12 Ry

     iteration # 15     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           119 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16597 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is        2.9 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          6.4E-13 Ry

     iteration # 16     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           119 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16596 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is        3.0 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          3.6E-13 Ry

     iteration # 17     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           119 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16583 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is        3.1 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          2.0E-13 Ry

     iteration # 18     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           119 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16591 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is        3.2 secs

     total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          1.0E-13 Ry

     iteration # 19     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
           119 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
         16591 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is        3.3 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 ( 16889 PWs)   bands (ev):

    -5.4907  -3.1408  -3.1406  -3.1406  -3.1403  -1.3266  -1.3262  -1.3262
    -1.3262  -1.3262  -1.3258  -0.5126  -0.5123  -0.5123  -0.5121   3.6095
     3.6096   3.6097   3.6097   3.6097   3.6099   5.2995   5.2999   5.3000
     5.3000   5.3000   5.3001   5.3002   5.3005   6.5306   6.5309   6.5312

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000

     highest occupied level (ev):     6.5312

!    total energy              =    -133.89470891 Ry
     estimated scf accuracy    <          6.3E-14 Ry

     The total energy is the sum of the following terms:
     one-electron contribution =      40.47028943 Ry
     hartree contribution      =      10.10928378 Ry
     xc contribution           =     -50.09560204 Ry
     ewald contribution        =    -134.37868008 Ry

     convergence has been achieved in  19 iterations

     Forces acting on atoms (cartesian axes, Ry/au):

     atom    1 type  1   force =     0.00018427    0.00000019   -0.00018427
     atom    2 type  1   force =    -0.00000461    0.00000004    0.00000461
     atom    3 type  1   force =     0.00002802    0.00001081   -0.00000623
     atom    4 type  1   force =     0.00000625   -0.00001086   -0.00002799
     atom    5 type  1   force =     0.00000623    0.00001081   -0.00002802
     atom    6 type  1   force =     0.00002799   -0.00001086   -0.00000625
     atom    7 type  1   force =     0.00001706    0.00000003   -0.00001706
     atom    8 type  1   force =    -0.00000772    0.00000003    0.00000772
     atom    9 type  1   force =    -0.00000684    0.00001276    0.00000684
     atom   10 type  1   force =    -0.00000690   -0.00001286    0.00000690
     atom   11 type  1   force =     0.00000595    0.00000004   -0.00000595
     atom   12 type  1   force =    -0.00003339    0.00000003    0.00003339
     atom   13 type  1   force =     0.00000595    0.00000004   -0.00000595
     atom   14 type  1   force =    -0.00003339    0.00000003    0.00003339
     atom   15 type  1   force =    -0.00009451   -0.00006115    0.00009451
     atom   16 type  1   force =    -0.00009436    0.00006093    0.00009436
     The non-local contrib.  to forces
     atom    1 type  1   force =     0.00025686    0.00000023   -0.00025686
     atom    2 type  1   force =    -0.00003586   -0.00000006    0.00003586
     atom    3 type  1   force =     0.00001016    0.00001007    0.00002582
     atom    4 type  1   force =    -0.00002586   -0.00001001   -0.00001004
     atom    5 type  1   force =    -0.00002582    0.00001007   -0.00001016
     atom    6 type  1   force =     0.00001004   -0.00001001    0.00002586
     atom    7 type  1   force =    -0.00001577   -0.00000010    0.00001577
     atom    8 type  1   force =     0.00000325   -0.00000006   -0.00000325
     atom    9 type  1   force =    -0.00000007   -0.00000286    0.00000007
     atom   10 type  1   force =    -0.00000011    0.00000285    0.00000011
     atom   11 type  1   force =    -0.00000298   -0.00000008    0.00000281
     atom   12 type  1   force =     0.00004284   -0.00000010   -0.00004258
     atom   13 type  1   force =    -0.00000281   -0.00000008    0.00000298
     atom   14 type  1   force =     0.00004258   -0.00000010   -0.00004284
     atom   15 type  1   force =    -0.00012826   -0.00017082    0.00012826
     atom   16 type  1   force =    -0.00012816    0.00017093    0.00012816
     The ionic contribution  to forces
     atom    1 type  1   force =     0.00065775    0.00000044   -0.00065775
     atom    2 type  1   force =    -0.00024060   -0.00000000    0.00024060
     atom    3 type  1   force =     0.00004689    0.00015138    0.00008922
     atom    4 type  1   force =    -0.00008922   -0.00015138   -0.00004689
     atom    5 type  1   force =    -0.00008922    0.00015138   -0.00004689
     atom    6 type  1   force =     0.00004689   -0.00015138    0.00008922
     atom    7 type  1   force =     0.00006216   -0.00000000   -0.00006216
     atom    8 type  1   force =    -0.00004385   -0.00000000    0.00004385
     atom    9 type  1   force =    -0.00000853   -0.00007063    0.00000853
     atom   10 type  1   force =    -0.00000854    0.00007063    0.00000854
     atom   11 type  1   force =    -0.00007918    0.00000002    0.00007915
     atom   12 type  1   force =     0.00021390   -0.00000008   -0.00021376
     atom   13 type  1   force =    -0.00007915    0.00000002    0.00007918
     atom   14 type  1   force =     0.00021376   -0.00000008   -0.00021390
     atom   15 type  1   force =    -0.00030155   -0.00051552    0.00030155
     atom   16 type  1   force =    -0.00030150    0.00051519    0.00030150
     The local contribution  to forces
     atom    1 type  1   force =    -0.00071331   -0.00000037    0.00071331
     atom    2 type  1   force =     0.00026975    0.00000002   -0.00026975
     atom    3 type  1   force =    -0.00002854   -0.00015007   -0.00011982
     atom    4 type  1   force =     0.00011967    0.00014999    0.00002857
     atom    5 type  1   force =     0.00011982   -0.00015007    0.00002854
     atom    6 type  1   force =    -0.00002857    0.00014999   -0.00011967
     atom    7 type  1   force =    -0.00003032    0.00000015    0.00003032
     atom    8 type  1   force =     0.00003311   -0.00000003   -0.00003311
     atom    9 type  1   force =     0.00000171    0.00008597   -0.00000171
     atom   10 type  1   force =     0.00000189   -0.00008609   -0.00000189
     atom   11 type  1   force =     0.00008790    0.00000001   -0.00008770
     atom   12 type  1   force =    -0.00028696    0.00000027    0.00028668
     atom   13 type  1   force =     0.00008770    0.00000001   -0.00008790
     atom   14 type  1   force =    -0.00028668    0.00000027    0.00028696
     atom   15 type  1   force =     0.00032633    0.00061324   -0.00032633
     atom   16 type  1   force =     0.00032649   -0.00061315   -0.00032649
     The core correction contribution to forces
     atom    1 type  1   force =    -0.00001705   -0.00000002    0.00001705
     atom    2 type  1   force =     0.00000212    0.00000000   -0.00000212
     atom    3 type  1   force =    -0.00000044   -0.00000056   -0.00000156
     atom    4 type  1   force =     0.00000156    0.00000056    0.00000043
     atom    5 type  1   force =     0.00000156   -0.00000056    0.00000044
     atom    6 type  1   force =    -0.00000043    0.00000056   -0.00000156
     atom    7 type  1   force =     0.00000100    0.00000001   -0.00000100
     atom    8 type  1   force =    -0.00000025    0.00000000    0.00000025
     atom    9 type  1   force =    -0.00000004    0.00000026    0.00000004
     atom   10 type  1   force =    -0.00000003   -0.00000026    0.00000003
     atom   11 type  1   force =     0.00000022    0.00000000   -0.00000021
     atom   12 type  1   force =    -0.00000312    0.00000001    0.00000311
     atom   13 type  1   force =     0.00000021    0.00000000   -0.00000022
     atom   14 type  1   force =    -0.00000311    0.00000001    0.00000312
     atom   15 type  1   force =     0.00000890    0.00001200   -0.00000890
     atom   16 type  1   force =     0.00000889   -0.00001201   -0.00000889
     The Hubbard contrib.    to forces
     atom    1 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    2 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    3 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    4 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    5 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    6 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    7 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    8 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    9 type  1   force =     0.00000000    0.00000000    0.00000000
     atom   10 type  1   force =     0.00000000    0.00000000    0.00000000
     atom   11 type  1   force =     0.00000000    0.00000000    0.00000000
     atom   12 type  1   force =     0.00000000    0.00000000    0.00000000
     atom   13 type  1   force =     0.00000000    0.00000000    0.00000000
     atom   14 type  1   force =     0.00000000    0.00000000    0.00000000
     atom   15 type  1   force =     0.00000000    0.00000000    0.00000000
     atom   16 type  1   force =     0.00000000    0.00000000    0.00000000
     The SCF correction term to forces
     atom    1 type  1   force =     0.00000002   -0.00000010   -0.00000002
     atom    2 type  1   force =    -0.00000002    0.00000008    0.00000002
     atom    3 type  1   force =    -0.00000005   -0.00000001    0.00000011
     atom    4 type  1   force =     0.00000010   -0.00000002   -0.00000006
     atom    5 type  1   force =    -0.00000011   -0.00000001    0.00000005
     atom    6 type  1   force =     0.00000006   -0.00000002   -0.00000010
     atom    7 type  1   force =    -0.00000001   -0.00000003    0.00000001
     atom    8 type  1   force =     0.00000002    0.00000012   -0.00000002
     atom    9 type  1   force =     0.00000009    0.00000003   -0.00000009
     atom   10 type  1   force =    -0.00000011    0.00000001    0.00000011
     atom   11 type  1   force =    -0.00000000    0.00000009   -0.00000000
     atom   12 type  1   force =    -0.00000005   -0.00000007   -0.00000006
     atom   13 type  1   force =     0.00000000    0.00000009    0.00000000
     atom   14 type  1   force =     0.00000006   -0.00000007    0.00000005
     atom   15 type  1   force =     0.00000007   -0.00000005   -0.00000007
     atom   16 type  1   force =    -0.00000007   -0.00000002    0.00000007

     Total force =     0.000347     Total SCF correction =     0.000000

     Writing all to output data dir ./tmp/scf.save/
 
     init_run     :      0.24s CPU      0.29s WALL (       1 calls)
     electrons    :      2.82s CPU      3.01s WALL (       1 calls)
     forces       :      0.06s CPU      0.07s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.05s CPU      0.06s WALL (       1 calls)
     wfcinit:atom :      0.00s CPU      0.00s WALL (       1 calls)
     wfcinit:wfcr :      0.04s CPU      0.05s WALL (       1 calls)
     potinit      :      0.02s CPU      0.03s WALL (       1 calls)
     hinit0       :      0.02s CPU      0.04s WALL (       1 calls)

     Called by electrons:
     c_bands      :      2.18s CPU      2.32s WALL (      19 calls)
     sum_band     :      0.27s CPU      0.29s WALL (      19 calls)
     v_of_rho     :      0.29s CPU      0.31s WALL (      20 calls)
     v_h          :      0.02s CPU      0.02s WALL (      20 calls)
     v_xc         :      0.28s CPU      0.31s WALL (      21 calls)
     mix_rho      :      0.08s CPU      0.08s WALL (      19 calls)

     Called by c_bands:
     init_us_2    :      0.09s CPU      0.10s WALL (      39 calls)
     init_us_2:cp :      0.09s CPU      0.10s WALL (      39 calls)
     cegterg      :      1.65s CPU      1.78s WALL (      19 calls)

     Called by sum_band:
     sum_band:wei :      0.00s CPU      0.00s WALL (      19 calls)
     sum_band:loo :      0.22s CPU      0.24s WALL (      19 calls)
     sum_band:buf :      0.00s CPU      0.00s WALL (      19 calls)
     sum_band:ini :      0.04s CPU      0.05s WALL (      19 calls)

     Called by *egterg:
     cdiaghg      :      0.08s CPU      0.09s WALL (      97 calls)
     cegterg:over :      0.09s CPU      0.11s WALL (      78 calls)
     cegterg:upda :      0.06s CPU      0.06s WALL (      78 calls)
     cegterg:last :      0.06s CPU      0.06s WALL (      44 calls)
     h_psi        :      1.37s CPU      1.47s WALL (      98 calls)
     g_psi        :      0.01s CPU      0.01s WALL (      78 calls)

     Called by h_psi:
     h_psi:calbec :      0.22s CPU      0.23s WALL (      98 calls)
     vloc_psi     :      0.89s CPU      0.98s WALL (      98 calls)
     add_vuspsi   :      0.25s CPU      0.26s WALL (      98 calls)

     General routines
     calbec       :      0.23s CPU      0.24s WALL (     102 calls)
     fft          :      0.09s CPU      0.11s WALL (     210 calls)
     ffts         :      0.01s CPU      0.01s WALL (      19 calls)
     fftw         :      0.87s CPU      0.96s WALL (    4242 calls)
 
     Parallel routines
 
     PWSCF        :      3.16s CPU      3.42s WALL

 
   This run was terminated on:  15:49:42   4Jul2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
